bazel/execution/service.go

```diff
@@ -77,10 +116,28 @@ func (s *executionServer) Execute(
 	// Marshal ExecuteResponse to protobuf.Any format
 	resAsPBAny, err := ptypes.MarshalAny(&res)
 	if err != nil {
-		return nil, fmt.Errorf("Failed to marshal ExecuteResponse as ptypes/any.Any: %v", err)
+		log.Errorf("Failed to marshal ExecuteResponse: %s", err)
+		return nil, fmt.Errorf("Failed to marshal ExecuteResponse as ptypes/any.Any: %s", err)
 	}
 
+	log.Info("ExecuteRequest completed successfully")
 	// Include the response message in the longrunning operation message
 	op.Result = &longrunning.Operation_Response{Response: resAsPBAny}
 	return &op, nil
 }
+
+func execReqToScoot(req *remoteexecution.ExecuteRequest) (result sched.JobDefinition, err error) {
+	if req == nil {
+		return result, fmt.Errorf("Nil execute request")
+	}
+
+	// This creates a hardcoded job for now
```

Comment:
Obvious, but could you add something slightly more explicit wrt what's left TODO